dunstctl

```diff
@@ -117,7 +117,20 @@ case "${1:-}" in
 		fi
 		;;
 	"is-paused")
-		property_get paused | ( read -r _ _ paused; printf "%s\n" "${paused}"; )
+		exit=false
+		if [ "${2:-}" = "-e" ]; then
```

Comment:
The only other command taking a flag uses a long form. So, maybe `--exit-code` might be a better name (or maybe allow for long and short forms here).